tutorials/kubeadm.md

```diff
@@ -3,26 +3,62 @@
+    echo "please install yq"                                                                                                                                                                                                                  
+    exit 1                                                                                                                                                                                                                                    
+fi                                                                                                                                                                                                                                            
+                                                                                                                                                                                                                                              
+if [ ! $(which kubeadm) ]; then                                                                                                                                                                                                               
+    echo "please install kubeadm"                                                                                                                                                                                                             
+    exit 1                                                                                                                                                                                                                                    
+fi                                                                                                                                                                                                                                            
+                                                                                                                                                                                                                                              
+kubeadm config print init-defaults --component-configs=KubeletConfiguration > "$KUBEADM_CONFIG"                                                                                                                                                      
+yq w -i "$KUBEADM_CONFIG" nodeRegistration.criSocket unix:///var/run/crio/crio.sock                                                                                                                                                                  
+yq w -i --doc 2 "$KUBEADM_CONFIG" cgroupDriver systemd
+```
+
+This will create a kubeadm config file to configure the kubelet to run with the correct defaults to be able to communicate with CRI-O.
```

Comment:
soft suggestion.
```suggestion
This will create a kubeadm configuration file to configure the kubelet to run with the correct defaults enabling it to communicate with CRI-O.
```